pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [tool.poetry]
 name = "cognite-sdk"
 
-version = "7.13.4"
+version = "7.13.15"
 description = "Cognite Python SDK"
```

Comment:
Should be 5, not 15 (guessing typo). Awesome addition btw!

Reply:
oops

Reply:
Lol, I see it was Anders that lead you astray on this one 😂